src/windows/winjector-glue.c

```diff
@@ -1,10 +1,20 @@
 #include "frida-core.h"
+#include "access-helpers.h"
```

Comment:
Should add a blank line before this one, as `frida-core.h` is the header of this source file, and `access-helpers.h` belongs to the internal includes (second group).